### PR TITLE
Issue #13402 - Fixed assorted bugs in FDCHP data product algorithms

### PR DESCRIPTION
**Pull request by sgfoote**

### CODE REVIEW

File: ion_functions/data/prs_functions.py [hunk 1]
```diff
@@ -821,16 +821,17 @@ def anchor_bin_raw_data_to_15s(time, data):
...
 
     # anchor time-centered bins by determining the start time to be half a bin
     # before the first 'anchor timestamp', which will be an integral number of
     # bin_durations after midnight.
     start_time = np.floor((time[0] - half_bin)/bin_duration) * bin_duration + half_bin
     # calculate elapsed time from start in units of bin_duration.
-    time_elapsed = (time - start_time)/bin_duration
+    elapsed_time = time
+    elapsed_time -= start_time
+    elapsed_time /= bin_duration
```
Comment (tlfish3638) [inline ion_functions/data/prs_functions.py:832]: I see the variable name change but curious why you altered the 1-line command to 3 lines. It doesn't look like you changed the result.

Comment (sgfoote) [reply ion_functions/data/prs_functions.py:832]: It's a memory footprint issue. You're absolutely right that the result is the same. But the multi-line version manipulates the structure in place.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# Release 2.4.5 2020-03-31
+
+Issue #13402 - reduce test_prs_functions memory footprint so tests run
+
+Issue #13402 - fixed assorted bugs in FDCHP data product algorithms
+
```
Comment (tlfish3638) [inline RELEASE_NOTES.md:7]: Please consolidate the 2 release entries into 1

**Review by tlfish3638 — APPROVED** [review]

File: RELEASE_NOTES.md (hunk 2 above)
Comment (sgfoote) [reply RELEASE_NOTES.md:7]: done

**Review by steinermg — APPROVED** [review]

**Pull request merged**